eval/eval.go

```diff
@@ -1676,7 +1694,7 @@ func (e *Interpreter) runINPUT() error {
 	var input string
 
 	if flag.Lookup("test.v") == nil {
```

Comment:
This code can be improved now; the intent was to allow some minimal testing even though we couldn't fake/stub INPUT-reading.

Your pull-request allows a real test to be made, so the runtime lookup of the test.v flag can go.  

If you update the test-case to cover this then I think everything looks good :)